docs/running-on-kubernetes.md

```diff
@@ -83,7 +83,7 @@ the cluster.
 ### Specifying input files
 
 Spark supports specifying JAR paths that are either on the submitting host's disk, or are located on the disk of the
-driver and executors. Refer to the [application submission](submitting-applications.html#advanced-dependency-management)
+driver and executors. Refer to the [application submission](submitting-applications.md#advanced-dependency-management)
```

Comment:
Sorry I missed this information! What about the hint that the namespace must be exists? Otherwise I would close this PR.

Thanks!